README.md

```diff
@@ -40,13 +40,21 @@ These scripts can all be run from the root level of the repo:
   - This will also run [Lerna](https://lerna.js.org/) `bootstrap` which allows us to have multiple packages within the same repo (a monorepo). Lerna installs all our dependencies and symlinks any cross-dependencies.
 
 - `yarn build`
-  - Compile/transpile/uglify everything and makes things release-ready.
-  - `yarn build:healthcare` to build the Healthcare.gov Design System
-  - `yarn build:medicare` to build the Medicare.gov Design System
-- `yarn build:storybook:docs && yarn build:docs`
-
-  - Builds the docs site statically
-
+  - Compiles everything and makes things release-ready
+  - Building is required to get TypeScript completion for the core package in child design system packages
+- `yarn build:{core,cmsgov,healthcare,medicare}`
+  - Builds a specific brand of the design system and its corresponding package
+- `yarn build:docs`
+  - Statically builds a copy of the doc site
+  - You can then serve it with `yarn serve:docs`
+- `yarn build:storybook`
```

Comment:
Is `yarn build:storybook:docs` still a thing? I see it in package.json, but not described here.

Reply:
As an aside, should `yarn build:storybook:docs` actually be `yarn build:storybook-docs` to match the testing command?

Reply:
It's only used by other scripts to build storybook specifically for the docs. It isn't building "storybook docs" but rather building "storybook" for the "docs" package (Gatsby site). It's not something people should have to manually run, so I didn't want to explain it here.